cli/ts/utils/storage.ts

```diff
@@ -40,7 +40,7 @@ export const storeContractAddress = (contractName: string, address: string, netw
 /**
  * Read a contract address from the local address store file
  * @param contractName - the name of the contract
- * @returns the contract address or a undefined it it does not exist
+ * @returns the contract address or a undefined it does not exist
```

Comment:
I think this meant to read: "if it does not exist" - could you please amend?

Reply:
fixed @ctrlc03